roles/fstat/tasks/main.yml

```diff
@@ -31,3 +31,15 @@
     name: fstat
     enabled: True
     state: started
+
+- name: Configure Cron tab centos
```

Comment:
So, I am nitpicking, but why is it called "Configure Cron tab centos" since it configure more than the centos job crontab ?